omod/src/main/java/org/openmrs/module/registrationapp/RegistrationAppUiUtils.java

```diff
@@ -61,7 +61,23 @@ public String getAttribute(Person person, String attributeTypeUuid) {
 		
 		return null;
 	}
-	
+	/**
+	 * @since 1.15.0
+	 * Gets the person Attribute name  for the specified person Attribute
```

Comment:
Does it get the name or value?

Reply:
Would be great to replace `for the specified person attribute` with `for the specified person attribute type uuid`

Reply:
The method gets the display value, i have changed the description to include the suggestions made above